promql/functions.go

```diff
@@ -31,18 +31,23 @@ import (
 
 // FunctionCall is the type of a PromQL function implementation
 //
-// vals is a list of the evaluated arguments for the function call.
-//    For range vectors it will be a Matrix with one series, instant vectors a
-//    Vector, scalars a Vector with one series whose value is the scalar
-//    value,and nil for strings.
+// vals is a list of the evaluated arguments for the function call. For range
+// vectors it will be a Matrix with one series, instant vectors a Vector,
+// scalars a Vector with one series whose value is the scalar value,and nil for
+// strings.
+//
 // args are the original arguments to the function, where you can access
-//    matrixSelectors, vectorSelectors, and StringLiterals.
-// enh.Out is a pre-allocated empty vector that you may use to accumulate
-//    output before returning it. The vectors in vals should not be returned.a
-// Range vector functions need only return a vector with the right value,
-//     the metric and timestamp are not needed.
+// matrixSelectors, vectorSelectors, and StringLiterals.
+//
+// enh.Out is a pre-allocated empty vector that you may use to accumulate output
+// before returning it. The vectors in vals should not be returned.
+//
+// Range vector functions need only return a vector with the right value, the
+// metric and timestamp are not needed.
+//
 // Instant vector functions need only return a vector with the right values and
-//     metrics, the timestamp are not needed.
+// metrics, the timestamp are not needed.
+//
```

Comment:
Sorry for the reformatting here. It was weirdly formatted before this commit, and somehow Go1.18 didn't like the orginal formatting and formatted this even more weirdly. So I manually formatted it "normally".